.nuke/build.schema.json

```diff
@@ -122,6 +154,8 @@
               "DetermineBuildMetadata",
               "DetermineProjectMetadata",
               "FullBuild",
+              "GenerateDependDBOutput",
+              "GenerateSbom",
```

Comment:
Should be fine? Autogenerated nuke content...

Reply:
This file is just a JSON schema for the parameter files. As we added new parameters with a build script update, the changes happen here as well. They don't really matter but if we do something build script related we can commit them as well.